Changelog.rst

```diff
@@ -1,5 +1,12 @@
 Changelog
 =========
+0.6.5 (2024-10-28)
+------------------
+- added support for `occurrences.download_citation` :issue:`158`
+- added support for `occurrences.download_sql` :issue:`157`
+- added support for `occurrences.download_describe` :issue:`142`
+- added support for `literature.search` :issue:`149`
```

Comment:
Could an example be added for this function?

Reply:
I found the examples, so please ignore https://github.com/gbif/pygbif/commit/f7e30eb036959cc9cb0cbb42f5a9db60f935eace#diff-924f18c08aa9b7c612d810239caad1790bc31d03468a55bbbf3d654f1f254fc7